lib/resolve-deps.js

```diff
@@ -1,76 +1,86 @@
 "use strict";
```

Comment:
More importantly, we ought to rename this file. It didn't make sense before, and it makes less sense now. `build-code.js` would be more informative.